src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Microsoft.AspNetCore.Razor.LanguageServer.Test.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(DefaultNetCoreTargetFramework)</TargetFrameworks>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework)-windows</TargetFrameworks>
```

Comment:
This is interesting, does Arcade provide the `DefaultNetCoreTargetFramework`?

Reply:
It's in [Versions.props](https://github.com/dotnet/razor-tooling/blob/main/eng/Versions.props#L28)

This is the change I am least confident of, especially in the linux ci leg (but I don't know what that does?!), but given its only a test project I thought it was worth the gamble.